x-pack/plugins/actions/server/routes/delete.ts

```diff
@@ -5,18 +5,12 @@
  * 2.0.
  */
 
-import { schema } from '@kbn/config-schema';
 import { IRouter } from '@kbn/core/server';
 import { ILicenseState } from '../lib';
 import { BASE_ACTION_API_PATH } from '../../common';
 import { ActionsRequestHandlerContext } from '../types';
 import { verifyAccessAndContext } from './verify_access_and_context';
-
-const paramSchema = schema.object({
-  id: schema.string({
-    meta: { description: 'An identifier for the connector.' },
-  }),
-});
+import { deleteConnectorRequestParamsSchemaV1 } from '../../common/routes/connector/apis/delete';
 
 export const deleteActionRoute = (
```

Comment:
maybe while we're in here we can rename to `deleteConnectorRoute` :)

Reply:
sure!

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/194227/commits/a89ceb1dd0cd6c222946ea6069ea5e3208e87da1